app/src/main/java/com/demoapp/dagger/AppModule.kt

```diff
@@ -29,5 +28,4 @@ class AppModule(private val application: Application) {
     fun provideAppContext(): Context {
         return application.applicationContext
     }
-
 }
```

Comment:
end line. Change this in your Android Studio settings to ensure end line in all files. 🚫